velox/dwio/parquet/tests/duckdb_reader/ParquetTableScanTest.cpp

```diff
@@ -36,6 +36,9 @@ class ParquetTableScanTest : public HiveConnectorTestBase {
   void SetUp() override {
     HiveConnectorTestBase::SetUp();
     registerParquetReaderFactory();
+    facebook::velox::aggregate::registerCountAggregate();
```

Comment:
This shouldn't be needed after adding registerAllAggregateFunctions call to OperatorTestBase.

Reply:
Yes I added registerAllAggregateFunctions call to OperatorTestBase.